src/enchcracker/EnchCrackerWindow.java

```diff
@@ -781,6 +815,16 @@ public void mouseExited(MouseEvent e) {}
 			// actual enchantment
 			playerSeed = (playerSeed * 0x5deece66dL + 0xb) & 0x0000_ffff_ffff_ffffL;
 
+			try {
+				int playerLevel = Integer.parseInt(levelTextField.getText());
+				if (timesNeeded != -1) {
+					playerLevel--;
+				}
+				playerLevel -= (chosenSlot + 1);
+				levelTextField.setText(Integer.toString(playerLevel));
+			} catch (NumberFormatException e) {
+				Log.info("Could not update player level text");
+			}
 			timesNeeded = -2;
```

Comment:
Wouldn't it be good to also set chosenSlot to -1 here?

Reply:
I was wanting to err on the safe side, but that code was pretty horrible with the -1 and -2 anyway, so I think after a cleanup on my part it won't make much of a difference.